.github/workflows/create-block.yml

```diff
@@ -14,13 +14,17 @@ concurrency:
 
 jobs:
     checks:
-        name: Checks w/Node.js ${{ matrix.node }} on ${{ matrix.os }}
+        name: Checks w/Node.js ${{ matrix.node.name }} on ${{ matrix.os }}
         runs-on: ${{ matrix.os }}
         if: ${{ github.repository == 'WordPress/gutenberg' || github.event_name == 'pull_request' }}
         strategy:
             fail-fast: false
             matrix:
-                node: ['20', '22.4']
+                node:
+                    - name: 20
+                      version: 20
+                    - name: 22
+                      version: 22.4
```

Comment:
Array/Object notation in yaml, as documented here:
https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs